examples/mock/cluster_test.go

```diff
@@ -1,4 +1,4 @@
-package test
+package examples
```

Comment:
FYI @lantoli This was developed as test but were put into examples package. 
See: https://go.dev/doc/tutorial/add-a-test